src/commands/metafrage.ts

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+// ========================= //
+// = Copyright (c) NullDev = //
+// ========================= //
+
+// Dependencies
+let parseOptions = require("minimist");
+
+/**
+ * Sends instructions on how to ask better questions
+ * @param {import("discord.js").Client} client
+ * @param {import("discord.js").Message} message
+ * @param {Array} args
+ * @param {Function} callback
+ * @returns {Function} callback
+ */
+exports.run = (client: any, message: any, args: any, callback: any) => {
+    // parse options
+    let options = parseOptions(args, {
+        "boolean": [
+            "english"
+        ],
+        alias: {
+            english: "e"
+        }
+    });
+
+    if (args.length === 0 || (args.length === 1 && options.english)){
```

Comment:
Komische Formatierung mit den geschweiften Klammern aber okay, wenn ESLint nicht meckert =D

Ansonsten verstehe ich nicht was du mit dem `args.length` check erzielen willst. Reicht doch auf `options.english` in l. 46 zu testen

Reply:
Der Check soll verhindern, dass Leute z.b. .metafrage -e -d -e -in -e -m -u -d -d -a oder so einen Blödsinn fabrizieren. Entweder gibt es kein arg oder exakt eines, nämlich -e. Aber stimmt, theoretisch reicht dafür auch der Check auf options.english. Wird angepasst.